### PR TITLE
Add jquery UI 1.13.1 images and fix collectstatic

### PR DESCRIPTION
**Pull request by drfb**
Django `collectstatic` fails for references inside css comments. Running collectstatic will produce an error:

```
The CSS file 'silk/lib/jquery-ui-1.13.1.min.css' references a file which could not be found:
  silk/lib/"images/ui-icons_555555_256x240.png"
```

Reference: https://code.djangoproject.com/ticket/21080

### CODE REVIEW

**Comment by joshuadavidthomas** [comment]
Thanks @drfb!

File: silk/static/silk/lib/jquery-ui-1.13.1.min.css [hunk 1]
```diff
@@ -1,7 +1,6 @@
...
...rContent=%23444444&iconColorActive=%23ffffff&bgImgOpacityShadow=0&bgImgOpacityOverlay=0&bgImgOpacityError=95&bgImgOpacityHighlight=55&bgImgOpacityContent=75&bgImgOpacityHeader=75&bgImgOpacityActive=65&bgImgOpacityHover=75&bgImgOpacityDefault=75&bgTextureShadow=flat&bgTextureOverlay=flat&bgTextureError=flat&bgTextureHighlight=flat&bgTextureContent=flat&bgTextureHeader=flat&bgTextureActive=flat&bgTextureHover=flat&bgTextureDefault=flat&cornerRadius=3px&fwDefault=normal&ffDefault=Arial%2CHelvetica%2Csans-serif&fsDefault=1em&cornerRadiusShadow=8px&thicknessShadow=5px&offsetLeftShadow=0px&offsetTopShadow=0px&opacityShadow=.3&bgColorShadow=%23666666&opacityOverlay=.3&bgColorOverlay=%23aaaaaa&fcError=%235f3f3f&borderColorError=%23f1a899&bgColorError=%23fddfdf&fcHighlight=%23777620&borderColorHighlight=%23dad55e&bgColorHighlight=%23fffa90&fcContent=%23333333&borderColorContent=%23dddddd&bgColorContent=%23ffffff&fcHeader=%23333333&borderColorHeader=%23dddddd&bgColorHeader=%23e9e9e9&fcActive=%23ffffff&borderColorActive=%23003eff&bgColorActive=%23007fff&fcHover=%232b2b2b&borderColorHover=%23cccccc&bgColorHover=%23ededed&fcDefault=%23454545&borderColorDefault=%23c5c5c5&bgColorDefault=%23f6f6f6
```
Comment (joshuadavidthomas) [inline silk/static/silk/lib/jquery-ui-1.13.1.min.css:4]: What's the reasoning behind removing this url?

**Review by joshuadavidthomas — CHANGES_REQUESTED** [review]

File: silk/static/silk/lib/jquery-ui-1.13.1.min.css (hunk 1 above)
Comment (joshuadavidthomas) [reply silk/static/silk/lib/jquery-ui-1.13.1.min.css:4]: Ah, after testing you are right, this comment is an issue.

**Review by joshuadavidthomas — APPROVED** [review]

**Comment by joshuadavidthomas** [comment]
Fixes #574

**Review by albertyw — APPROVED** [review]

**Comment by albertyw** [comment]
Thanks for the fix.  I'll merge and make a new release shortly.

**Pull request merged**